core/string/sub_spec.rb

```diff
@@ -396,7 +396,7 @@
   end
 
   it "ignores non-String keys" do
-    "hello".sub(/(ll)/, 'll' => 'r', :ll => 'z').should == "hero"
+    "hello".sub(/(ll)/, 'll' => 'r', ll: 'z').should == "hero"
```

Comment:
@kotp It's two times the letter L, downcase, not eleven :wink:.
The CI would definitely have complained if there was such an error.